docs/index.md

```diff
@@ -15,6 +15,7 @@ The model definitions and the train loop are available in the [Larq Zoo reposito
 | -------------------------------------------- | -------------- | -------------- | ---------- | -------- |
 | [Binary AlexNet](/models/api/#binaryalexnet) | 36.30 %        | 61.53 %        | 61 859 192 | 7.49 MB  |
 | [Bi-Real Net](/models/api/#birealnet)        | 58.01 %        | 80.27 %        | 11 736 232 | 4.04 MB  |
+| [ResNetE18](/models/api/#resnete18)          | 58.26 %        | 80.84 %        | 11 699 368 | 4.03 MB  |
 | [XNOR-Net](/models/api/#xnornet)             | 44.96 %        | 69.18 %        | 62 396 768 | 22.81 MB |
```

Comment:
It's good to see you have beaten the paper accuracy, but do you think that it could go higher? Looking at it from this table it looks pretty much identical to Bi-RealNet in terms of accuracy, params and model size

Reply:
I reran the [example](https://larq.dev/models/examples/#evaluate-bi-real-net-with-tensorflow-datasets) and updated the table with the exact accuracies (58.32 % Top-1 and 80.79 % Top-5).

I didn't spend any time tuning this network, so I think there is still room for improvement, but that's not a high priority for now. I think it is still a valuable addition since the network is conceptionally simpler than BiRealNet (just uses STE) and trains in less epochs to the claimed accuracy.